src/Dialogs/StripeDialog.vala

```diff
@@ -94,16 +94,48 @@ public class AppCenter.Widgets.StripeDialog : Granite.Dialog {
         primary_label.xalign = 0;
 
         var secondary_label = new Gtk.Label (
-            _("This is a one time payment. Your email address is only used to send you a receipt.") +
-            " <a href=\"https://stripe.com/privacy\">%s</a>".printf (_("Privacy Policy"))
+            _("This is a one time payment. You can also choose to pay a more affordable price.")
```

Comment:
I want to avoid accidentally implying this is unaffordable and also consider that someone might want to pay more than the sticker price. Maybe something like:

```suggestion
            _("This is a one time payment suggested by the developer. You can also choose your own price.")
```

or maybe

```suggestion
            _("This is a one time payment suggested by the developer. You can choose to pay it or select your own price.")
```

or just something along those lines